tests/testthat/test-carehomes-multistrain.R

```diff
@@ -580,7 +580,7 @@ test_that("Can calculate Rt with a second less infectious variant", {
 test_that("Can calculate Rt with a second more infectious variant", {
   ## Seed with 10 cases on same day as other variant
   p <- carehomes_parameters(sircovid_date("2020-02-07"), "england",
-                            strain_transmission = c(1, 10),
+                            strain_transmission = c(1, 5),
```

Comment:
With 10, this now causes the epidemic to die out within this time frame. And once we have 0 infectives, the multistrain version yields NA for R, so the test would fail. Hence I have reduced it to 5, so that the epidemic doesn't die out within the time frame.